### ISSUE TITLE
TAB key highlighting not working for Dashboard left side menu

### ISSUE THREAD

**Issue by jlhardes** [samvera/hyrax#5637]
### Descriptive summary

Tabbing through Dashboard is working for header and main area of Dashboard but left-side menu options are not all visibly highlighted with TAB key. As an example, cannot see that Works option from left-side menu is selected when tabbing, which is specific feature test for release under Accessibility.

### Rationale

All links should be visibly accessible using the TAB key in both the end user view and the Dashboard.

### Expected behavior

Tabbing through Dashboard will go through header links, then down all left-side menu links, then into main area of the Dashboard through all links there with highlight showing around each link when it is tabbed.

### Actual behavior

Can see breadcrumbs highlighted when tabbed to in main area of Dashboard and can get to Works list that way but left side menu is not visibly tab-able and Works menu item is one of several left side menu items that does not look reachable.

### Steps to reproduce the behavior

1. Navigate to the Dashboard screen
2. Using TAB key to cycle through links, try to select "Works" from the left-side of the Dashboard under the “Repository Contents” heading - this link is not highlighted along with other left-side menu links and is difficult to select

### Related work

Other tickets might be related to this 

#5638 - Dashboard left side menus don't stay expanded
https://github.com/samvera/hyrax/issues/5639 - Skip to Content link not viewable in Dashboard).

**Comment by bwatson78** [comment]
Able to select Works via `Tab`-bing. The Docker container has been building quite reliably recently. I'm pretty confident that bringing up the container will confirm this.

**Comment by jlhardes** [comment]
Testing locally with `bl7boot4rails6` branch shows this working correctly. Thanks!
